projects/js-packages/publicize-components/src/components/media-section/index.js

```diff
@@ -40,34 +44,53 @@ const ADD_MEDIA_LABEL = __( 'Set Social Image', 'jetpack' );
 const REPLACE_MEDIA_LABEL = __( 'Replace Social Image', 'jetpack' );
 const REMOVE_MEDIA_LABEL = __( 'Remove Social Image', 'jetpack' );
 
-const ALLOWED_MEDIA_TYPES = [ 'image/jpeg', 'image/png' ];
-
 /**
  * Wrapper that handles media-related functionality.
  *
  * @returns {object} The media section.
  */
 export default function MediaSection() {
+	const [ showNotice, setShotNotice ] = useState( false );
 	const { attachedMedia, updateAttachedMedia } = useAttachedMedia();
+	const { connections } = useSocialMediaConnections();
```

Comment:
I think we have the enabled connections coming back from the hook.

```suggestion
	const { enabledConnections } = useSocialMediaConnections();
```

Reply:
Yeah, we have but that one maps it after filtering, and only gives back the ID-s. The same goes for `skippedConnections`. We could change that, but I saw that it is used in some places.

Reply:
Ah good point! I think [it might only be used here](
https://github.com/Automattic/jetpack/tree/e88dcaf15cf2004fe1813566b05fa733f25ce2fd/projects/js-packages/publicize-components/src/components/form/index.js#L45), and that is to check the length, so we could return the full connection.

I'm worried that those arrays aren't memoized anyway, so they'll be changing on every render. That will be happening here too with the array filter, because that will return a new array reference.

Reply:
I changed the declaration of it, now we can use it from the hook